### PR TITLE
feat: add lambdos

### PR DESCRIPTION
**Pull request by manumena**

### CODE REVIEW

**Comment by menduz** [comment]
Please pick a name that doesn't lead to confusions for the image name & repo. It may lead to human typing/reading errors lambdas and lambdos are very close with eachother

File: docker-compose.yml [hunk 1]
```diff
@@ -178,6 +178,20 @@ services:
     volumes:
       - "lambdas_server_storage:/app/storage/lambdas"
 
+  lamb2:
+    image: quay.io/decentraland/lamb2:${LAMB2_DOCKER_TAG:-latest}
+    working_dir: /app
+    restart: always
+    environment:
+      - HTTP_SERVER_PORT=7272
+      - CONTENT_SERVER_ADDRESS=https://peer.decentraland.org/content
```
Comment (menduz) [inline docker-compose.yml:187]: Should this point to the internal content server?
```suggestion
      - CONTENT_SERVER_ADDRESS=http://content
```

Comment (manumena) [reply docker-compose.yml:187]: Yessss, good catch agus. Thanks

Comment (menduz) [reply docker-compose.yml:187]: Also secured metrics endpoints plz 🙏

**Review by hugoArregui — APPROVED** [review]

**Pull request merged**